Relationships.Infrastructure/EventBus/EventBusConfiguration.cs

```diff
@@ -6,7 +6,12 @@ public class EventBusConfiguration
     public string RabbitMQUsername { get; set; }
     public string RabbitMQPassword { get; set; }
 
-    public bool AzureServiceBusEnabled { get; set; }
     public int ConnectionRetryCount { get; set; }
     public string SubscriptionClientName { get; set; }
+
+    public string ProjectId { get; set; }
+    public string TopicName { get; set; }
```

Comment:
Maybe you should a a `GcpPubSub` prefix so that it is obvious that these properties are only available for GCP.

```suggestion
    public string GcpPubSubProjectId { get; set; }
    public string GcpPubSubTopicName { get; set; }
```

If some day we need the properties for other vendors as well, we can still rename them.

Of course this leads to very ugly property names, but I would prefer that over the possible confusion of the current solution.